game-of-life-native/Game of Life/views/UIBoard.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  UIBoardView.swift
+//  Game of Life
+//
+//  Created by Tobias Heine on 07.02.18.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import UIKit
+import KotlinGameOfLife
+
+class UIBoard: UIView, KGOLBoardView {
```

Comment:
`KGOLBoardView` see [here](https://github.com/novoda/spikes/pull/364/files#diff-be5420527526d13052a073d273bb3187) for the kotlin implementation.